cmd/terminal.go

```diff
@@ -235,11 +296,117 @@ func (c *TerminalCmd) stopShell(conn *websocket.Conn) error {
 	return nil
 }
 
+func (c *TerminalCmd) record() {
+	f, err := os.Create(c.recordFile)
+	if err != nil {
+		log.Err(fmt.Sprintf("Can't create recording file: %s: %s", c.recordFile, err.Error()))
+	}
+	defer f.Close()
+
+	fz := gzip.NewWriter(f)
+	defer fz.Close()
+
+	data := TerminalRecordingHeader{
+		Version: terminalRecordingVersion,
+	}
+	copy(data.DeviceID[:], []byte(c.deviceID))
+	copy(data.DateTime[:], time.Now().String())
```

Comment:
another nitpick, `UTC()` maybe? more as a question than a request for change.

Reply:
I don't know. As this is supposed to be a local feature (save to local file, playback from local file) I think it will be more readable with local time so I wanted to save the string.

If you want, what I can do is save a unix timestamp and then just convert it to local time on playback (to display it nicely to the user).

Reply:
> I don't know. As this is supposed to be a local feature (save to local file, playback from local file) I think it will be more readable with local time so I wanted to save the string.

makes sense.

> If you want, what I can do is save a unix timestamp and then just convert it to local time on playback (to display it nicely to the user).

this is the top-of-the-line solution! I like it! (but you call, lets not make it too complex, so I am fine if you decide to keep it as is)

Reply:
It is very simple. Coming up in next rework.